src/formats.py

```diff
@@ -90,5 +90,7 @@ def get_formatter(definition, hostname, log_name, log_token):
 def _sanitize_syslog_name(original):
     """Replaces invalid characters from syslog name entry.
     """
-    originalBytes = original.encode('utf-8')
-    return originalBytes.decode('utf8', 'ignore').replace(' ', '_')
+    if isinstance(original, bytes):
```

Comment:
This would work, but I am thinking if this is to to sanitize the syslog name, can read the syslog name as unicode instead of doing the conversion here. 

Referencing this:
https://docs.python.org/2/howto/pyporting.html#text-versus-binary-data